createx/src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../Button/index';

```

Comment:
Always try to use nested root imports not direct imports.

```suggestion
import Button from '../components;

```